torch/fx/graph.py

```diff
@@ -4,6 +4,7 @@
 import torch.utils._pytree as pytree
 from . import _pytree as fx_pytree
 from ._compatibility import compatibility
+from torch._C import _NodeIter
```

Comment:
Do you think we ought to preserve the old Python implementation, similar to how some CPython modules have both pure Python and C versions?

Reply:
When would we use the pure python versions? I can implement classes that do exactly the same in python but I assume we will keep moving parts of fx to c++

Reply:
I assume it's more as a reference implementation / spec / testing utility. IDK, I don't feel strongly about it.